Features/links.feature

```diff
@@ -15,7 +15,18 @@ Voor tijdreisvragen endpoints:
  - geldig op en beschikbaar op in vraag = geldig op en beschikbaar op in links van gerelateerde objecten
 
  Paginering of andere query parameters zijn bij gerelateerde links niet van toepassing, die zijn alleen van toepassing voor self link
+ 
+ Voor endpoints die collecties opleveren:
+  opbouw self link van de collectie:
+  Er wordt exact overgenomen wat er door een client is opgegeven.
+  Geeft een client meer parameters op dan worden die exact overgenomen.
+  Geeft een client bepaalde parameters niet op en wordt een default gebruikt dan komen die niet in de self link terug.
 
+  opbouw self link embedded resource
+  De self links in de embedded resources verwijzen naar het specifieke voorkomen het object
```

Comment:
Dit is niet correct. Ook bij andere endpoints dan bij /lvc worden collecties teruggegeven, het is dan niet per definitie zo, dat de self link van embedded resources bij die endpoints dan naar een voorkomen verwijzen. Voorkomens moeten altijd alleen bij /lvc endpoints worden teruggegeven.